python/lsst/ts/watcher/rules/mt_air_compressors_pressure.py

```diff
@@ -0,0 +1,162 @@
+                name="MTAirCompressor",
+                index=index,
+                callback_names=[],
+                poll_names=["tel_analogData"],
+            )
+            for index in SAL_INDICES
+        ]
+        super().__init__(
+            config=config,
+            name="MTAirCompressorsEnabled",
+            remote_info_list=remote_infos,
+        )
+        self.threshold_handler = watcher.ThresholdHandler(
+            warning_level=getattr(config, "warning_level", None),
+            serious_level=getattr(config, "serious_level", None),
+            critical_level=getattr(config, "critical_level", None),
+            warning_msg=getattr(config, "warning_msg", None),
+            serious_msg=getattr(config, "serious_msg", None),
+            critical_msg=getattr(config, "critical_msg", None),
+            hysteresis=config.hysteresis,
+            big_is_bad=False,
+            value_name="linePressure",
+            units="mbar",
+            value_format="0.4f",
+        )
+        super().__init__(
+            config=config,
+            rule_name="MTAirCompressorPressure",
+        )
+
+    @classmethod
+    def get_schema(cls):
+        schema_yaml = """
+$schema: http://json-schema.org/draft-07/schema#
+description: Configuration for MTAirCompressorPressure rule.
+type: object
+properties:
+  warning_level:
+    description: >-
+        The pressure (mbar) below which a warning alarm is issued.
+        Omit for no warning alarm.
+    type: number
+  serious_level:
+    description: >-
+        The pressure (mbar) below which a serious alarm is issued.
+        Omit for no serious alarm.
+    type: number
+  critical_level:
+    description: >-
+        The pressure (mbar) below which a critical alarm is issued.
+        Omit for no critical alarm.
+        Typically 8200.
+    type: number
+  hysteresis:
+    description: >-
+        The amount by which pressure (mbar) must decrease below
+        a severity level before alarm severity is decreased.
+    type: number
+    default: 10
+  poll_interval:
+    description: Time delay between polling updates (second).
+    type: number
+    default: 60
+  max_data_age:
+    description: >-
+      Maximum age of data that will be used (seconds). If all
+      pressure data is older than this, go to SERIOUS severity.
+    type: number
+    default: 120
+required:
+  - hysteresis
+  - poll_interval
+  - max_data_age
+additionalProperties: false
+       """
+        return yaml.safe_load(schema_yaml)
+
+    def setup(self, model):
+        self.topics = [
+            model.remotes[("MTAirCompressor", index)].tel_analogData
+            for index in SAL_INDICES
+        ]
+
+    def compute_alarm_severity(self):
+        data_list = [topic.get() for topic in self.topics]
+        pressure_index_list = [
+            (data.linePressure, data.sal_index)
+            for data in data_list
+            if data is not None
+        ]
+        if not pressure_index_list:
```

Comment:
You construct an array which can hold two "None". not array doesn't mean it does not contain None. I believe this condition shall be if not(None in pressure_index_list):

Reply:
No, the "if data is not None" at line 144 above prevents the list from containing None.